### PR TITLE
[stable/traefik] Option to specify minimum TLS version for https entrypoint

### PR DESCRIPTION
**Pull request by jacobsvante**
**What this PR does / why we need it**:

This option is needed to disable old and insecure TLS versions.

@krancour @emilevauge @dtomcej @Idez

### CODE REVIEW

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.29.2
+version: 1.29.3
```
Comment (timstoop) [inline stable/traefik/Chart.yaml]: Please bump the minor version of this chart, as you're adding a new feature.

**Review by timstoop — CHANGES_REQUESTED** [review]

File: stable/traefik/Chart.yaml (hunk 1 above)
Comment (jacobsvante) [reply stable/traefik/Chart.yaml]: @timstoop Done

File: stable/traefik/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.30.0
+version: 1.31.0
```
Comment (krancour) [inline stable/traefik/Chart.yaml]: If you make this 1.32.0 (something else just got merged ahead of this), then this will LGTM.

**Comment by jacobsvante** [comment]
@krancour I've bumped the version as per your comment

**Comment by krancour** [comment]
Needs rebase. There's a conflict over app version now. A recent PR bumped Traefik to 1.6.2.

**Comment by jacobsvante** [comment]
@krancour I think I read somewhere that we shouldn't do rebasing on PRs for kubernetes/charts?

Anyways I updated the Chart.yaml. Lgtm?

**Comment by timstoop** [comment]
Correct, merge from master is the way to do it, like you did. (I used to tell people to rebase as well, until it was pointed out to me that's not convenient.)

**Pull request merged**